app/models/user.rb

```diff
@@ -56,11 +56,11 @@ def mentor?
   end
 
   def npq?
-    npq_profiles.any?(&:active?)
+    npq_profiles.any?
   end
 
   def participant?
-    early_career_teacher? || mentor?
+    participant_profiles.any?
```

Comment:
This is a change in behaviour

Reply:
Original implementation restored. However, this method returns `false` for NPQ participant - which is not correct with the language

Reply:
Agreed, but we should change the language, not the behaviour